packages/pigeon/example/README.md

```diff
@@ -1,48 +1,13 @@
 <?code-excerpt path-base="app"?>
 # Pigeon Examples
 
+For a quick start guide to Pigeon, see the [QuickStart guide](../README.md#Quickstart). For language-specific guides to instantiate code generated by Pigeon, continue with
+this guide.
+
 The examples here will cover basic usage. For a more thorough set of examples,
 check the [core_tests pigeon file](../pigeons/core_tests.dart) and 
 [platform test folder](../platform_tests/) ([shared_test_plugin_code](../platform_tests/shared_test_plugin_code/) and [alternate_language_test_plugin](../platform_tests/alternate_language_test_plugin/) especially).
 
-## Invocation
```

Comment:
This removed sections seems like a much quicker quick-start than the new section that's called a quick start.

Reply:
It was shorter, that's true, but at least for me it was definitely not quicker. I also said this in an earlier comment, but in my first experience using Pigeon, I found myself searching through sample code to fill in large mental gaps in how everything was supposed to work.